packages/paste-core/components/popover/src/PopoverContainer.tsx

```diff
@@ -16,17 +15,16 @@ export interface PopoverContainerProps extends NonModalDialogPrimitivePopoverIni
   state?: PopoverStateReturn;
 }
 
-const BasePopoverContainer = React.forwardRef<HTMLDivElement, PopoverContainerProps>(
-  ({children, gutter, placement, state, ...initialState}, ref) => {
-    const popover = state || useNonModalDialogPrimitiveState({gutter, modal: true, placement, ...initialState});
-    return (
-      <PopoverContext.Provider value={popover}>
-        <Box ref={ref}>{children}</Box>
```

Comment:
What does removing the `ref` here do exactly?

Reply:
All our components forward refs, so I think it was just doing what other components do. But this isn't a standard component, it's similar to Theme.Provider and is just a React Context.  

Removing or adding ref would ordinarily breaking change, but since this isn't a standard component and ref here is arguably useless, and since reporting shows no one is passing ref to this --- it should be harmless to remove and mark as a minor change.